package.json

```diff
@@ -21,9 +21,12 @@
   "homepage": "https://github.com/GoogleChrome/proxy-polyfill#readme",
   "devDependencies": {
     "chai": "^4.1.2",
+    "google-closure-compiler": "^20180204.0.0",
```

Comment:
could this be a peer dependency? something we've been discussing internally for Closure.

Reply:
It is needed for Travis to run the build before the test. I could remove the pretest script, but I believe it is keep it this way.